packages/next/build/webpack/plugins/font-stylesheet-gathering-plugin.ts

```diff
@@ -52,9 +53,18 @@ export class FontStylesheetGatheringPlugin {
   gatheredStylesheets: Array<string> = []
   manifestContent: FontManifest = []
   isLikeServerless: boolean
+  optimizeFonts: any
 
-  constructor({ isLikeServerless }: { isLikeServerless: boolean }) {
+  constructor({
+    isLikeServerless,
+    optimizeFonts,
+  }: {
+    isLikeServerless: boolean
+    optimizeFonts: any
```

Comment:
Can we constrain this type away from `any`? e.g. with an object that contains `experimentalFontFallbacks` and `inlineFonts` specifically?